src/main/java/com/revature/model/CurriculumTopic.java

```diff
@@ -0,0 +1,36 @@
+package com.revature.model;
+
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.MapsId;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class CurriculumTopic {
+	
+
+	    @EmbeddedId
+	    CurriculumTopicKey curriculumTopicKey;
+
+	    @ManyToOne
+	    @MapsId("curriculum_id")
+	    @JoinColumn(name = "curriculum_id")
+	    private int curriculumId;
+
+	    @ManyToOne
+	    @MapsId("topic_id")
+	    @JoinColumn(name = "topic_id")
+	    private int topic_id;
```

Comment:
Best practice is to name fields with camelCasing (not snake_casing) in Java.  Please change this to `topicId`. 

If you want the column name to be `topic_id` in the DB, you can annotate this field with `@Column(name = "topic_id")`